landing-page/styles.css

```diff
@@ -0,0 +1,122 @@
+*{
+  margin:0;
+  padding:0;
+}
+body{
+  margin:0;
+  font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;
+  font-size: 17px;
+  line-height:1.6;
+}
+#simbol {
+  text-align: center;
+  color: black;
+  font: italic small-caps bold 30px Georgia, Garamond, serif;
+  background: limegreen;
+  margin: 10px;
+  border: 7px solid purple;
+  padding: 10px;
+
+}
+#showcase {
+  background-image:url('https://www.designcuts.com/wp-content/uploads/2018/11/30-Spring-Watercolor-Backgrounds-strip5.jpg');
+  background-size: cover;
+  background-position: center;
+  height:100vh;
+  display:flex;
+  flex-direction:column;
+  justify-content:center;
+  align-items:center;
+  text-align:center;
+  padding:0 20px;
+}
+#showcase h1 {
+  font-size:30px;
+  line-height:1.2;
+  }
+#showcase p{
+  font-size:20px;
+}
+#showcase .button1 {
+  font-size:18px;
+  text-decoration:none;
+  color: black;
+  border: black 1px solid;
+  padding:10px 20px;
+  border-radius:10px;
+  margin-top:15px;
+}
+#showcase .button1:hover {
+  background: darkcyan;
+  color:#fff;
+}
+
+#section-a {
+  padding:30px;
+  color:#fff;
+  text-align:center;
+}
+
+.button2 {
+  font-size:18px;
+  text-decoration:none;
+  color: black;
+  border: black 1px solid;
+  padding:10px 20px;
+  border-radius:10px;
+  margin-top:20px;
+}
+
+.button2:hover {
+  background: darkcyan;
+  color:#fff;
+}
+
+#section-b {
+  display:flex;
+  padding:0px;
+  justify-content:center;
+  align-items:center;
+  text-align:center;
+  
+}
+ .column1 {
+  background-image: 
+  url('https://www.designcuts.com/wp-content/uploads/2018/11/30-Spring-Watercolor-Backgrounds-strip5.jpg');
+  padding:30px 30px;
+  border:30px;
+  margin:0px;
+}
+.column2 {
+  background-image: 
+  url('https://4vector.com/iraw/free-vector-green-natural-blur-the-background-02-vector_016894_2.jpg');
+  padding:30px 30px;
+  border:30px;
+  margin:0px;
+}
+.column3 {
+  background-image: 
+  url('https://www.fg-a.com/wallpapers/red-abstract-background-1280.jpg');
+  padding:30px 30px;
+  border:30px;
+  margin:0px;
+}
```

Comment:
I like your instinct to clearly name your column elements! But do you notice how you have the exact same properties defined for each of them, except for the background-image url? What you could do instead is have a `column` class with all the styles you want to share across every column and put it on all your column elements in addition to the distinct name (so each element would be something like `<div class="column column3"> and the styles would be 
``` 
.column {
  padding: 30px;
}

.column3 {
   background-image: url('https://www.fg-a.com/wallpapers/red-abstract-background-1280.jpg');
}
```

The margin and border rules aren't doing anything (the margin because it is already inheriting that, and the border because you need to define more values to have it apply: https://developer.mozilla.org/en-US/docs/Web/CSS/border